### PR TITLE
select all button 

### LINKED ISSUE #7502: checkbox select all
select all is not working properly in /dashboard
Screenshot from 2020-02-15 21-30-15

Thanks!!

### PR DESCRIPTION
**Pull request by keshavsethi**
Fixes #7502
Select all issue in dashboard is solved by adding a small function of click all. now it is working fine as shown:
Screenshot from 2020-02-18 00-32-49
@cesswairimu @VladimirMikulic @SidharthBansal 
Thanks!

### CODE REVIEW

**Comment by nstjean** [comment]
Hi! While this does work, @sssash18 is working on a larger fix for this drop-down menu https://github.com/publiclab/plots2/pull/7348

**Comment by keshavsethi** [comment]
@VladimirMikulic , Thanks I will correct this.

**Comment by keshavsethi** [comment]
@VladimirMikulic 
Screenshot from 2020-02-18 02-37-37

File: app/assets/javascripts/dashboard.js [hunk 1]
```diff
@@ -45,7 +45,13 @@
         $('.node-type-' + type).prop('checked', checked);
  
       }
-
+      $('.node-type-all').click(function () {
+        $('.node-type').prop('checked', this.checked);
+    });
+    $('.node-type').change(function () {
```
Comment (VladimirMikulic) [inline app/assets/javascripts/dashboard.js]: This should be after the `.node-type`(in the same block). They are both related to checking/unchecking boxes functionality.

Comment (SidharthBansal) [reply app/assets/javascripts/dashboard.js:51]: @Keshavsethi kindly reply

Comment (keshavsethi) [reply app/assets/javascripts/dashboard.js:51]: Sure, I will make changes.

**Comment by cesswairimu** [comment]
restarting travis

**Review by jywarren — APPROVED** [review]

**Pull request merged**